.release-plan.json

```diff
@@ -14,7 +14,7 @@
           "reason": "Appears in changelog section :bug: Bug Fix"
         }
       ],
-      "pkgJSONPath": "./package.json"
+      "pkgJSONPath": "./ember-set-helper/package.json"
```

Comment:
On next release this should cause the addon package to be released instead of the whole project